packages/svelte/test/specimens/external-with-values.html

```diff
@@ -0,0 +1,13 @@
+<link rel="stylesheet" href="./external-with-values.css">
+
+<div {style}>
+    Support standalone values 2018
+</div>
+
+<script>
+    export default {
+        computed : {
+            style : () => `color: css.primary`
```

Comment:
```suggestion
            style : () => `color: ${css.primary}`
```

So that when replaced it'll become `style : () => `color: ${"#BEEFED"}` to help keep things accurate, otherwise it'd be `color: "#BEEFED"` which isn't the intended result.